### PR TITLE
Update issue template to add direct links to Signal docs and community

### PR DESCRIPTION
**Pull request by EwoutH**
### Contributor checklist
_No functional code change, not relevant_

- - - - - - - - - -

### Description
I updated the issue template using the new [issue template syntax](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser). With this PR, if users click the green _New issue_ button on the Issue tab, they will see the following screen first:

Screenshot_328

I hope this helps stop the endless flow of issues. I allowed maintainers to modify this PR, use the code in any way you like. The same `config.yml` file can be used in all other repositories.

Tagging @alan-signal and @greyson-signal because you might find this very useful!

### CODE REVIEW

**Comment by EwoutH** [comment]
A similar PR just got merged into the iOS repo, open an issue [over there](https://github.com/signalapp/Signal-iOS/issues) to see how it looks and what it feels like!

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 📃 User documentation
+    url: https://support.signal.org/
+    about: Search for any questions you have about Signal
+  - name: 📖 Developer documentation
+    url: https://signal.org/docs/
+    about: Official Signal deveoper documentation
```
Comment (rmueller83) [inline .github/ISSUE_TEMPLATE/config.yml:8]: typo in developer

Comment (EwoutH) [reply .github/ISSUE_TEMPLATE/config.yml]: Good catch, thanks!

**Comment by greyson-signal** [comment]
Hi there, thank you! I've updated some of the strings, but otherwise this commit will be in 5.2.1. Thanks!

**Review by Relluem94 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    url: https://support.signal.org/
+    about: Search for any questions you have about Signal
+  - name: 📖 Developer documentation
+    url: https://signal.org/docs/
+    about: Official Signal developer documentation 
+  - name: 💬 Community support
+    url: https://community.signalusers.org/c/support/
+    about: Feel free to ask anything
+  - name: ✨ Feature request
+    url: https://community.signalusers.org/c/feature-requests/
```
Comment (Rhahkeem) [inline .github/ISSUE_TEMPLATE/config.yml:13]: Should this be something like:
```suggestion
    url: https://community.signalusers.org/c/feature-requests/android-feature-requests/18
```
To bring them directly to the android tag within that topic.

This is great by the way!

**Pull request closed without merging**